src/Components/GetArticleComments.jsx

```diff
@@ -15,7 +16,7 @@ useEffect(() => {
         console.error(err)
         setIsLoading(false)
       });
-  }, []);
+  }, [comments]);
```

Comment:
The comments in this dependancy array are causing an infinite loop. I placed a console log in the useEffect on gitpod and get this:
![image](https://github.com/user-attachments/assets/dc24fedc-4324-48bf-b270-785bede2b237)